demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_tooltip.scss

```diff
@@ -50,7 +50,7 @@ $tooltip-width: 400px;
 //  Tooltip styles used by the Js and Vue implementation
 //  Templates:
 //  * DemosPlanCoreBundle/Resources/client/js/lib/Tooltips.js
-//  * DemosPlanCoreBundle/Resources/client/js/VueConfigCore.js
+//  * DemosPlanCoreBundle/Resources/client/js/InitVue.js
```

Comment:
Templates are located within `client/js/lib/core/libs/Tooltips.js:41` and `@demos-europe/demosplan-ui/src/directives/Tooltip/Tooltip.js:15`

Reply:
changed in 6852b8a27abb59355b169831fc0863f1736e9ab6